### PR TITLE
82 add logging on debug false

### PR DESCRIPTION
**Pull request by Mishawaka**
[82](https://trello.com/c/s628fgJo/82-backendlog-add-loggin-on-debug-false)

### CODE REVIEW

File: server/config/settings.py [hunk 1]
```diff
@@ -94,22 +94,26 @@
...
 		},
-		'loggers': {
-				'django': {
-						'handlers': ['file'],
-						'level': 'DEBUG',
-						'propagate': True,
-				},
+		'console': {
+		'level': 'INFO',
+		'class': 'logging.StreamHandler',
```
Comment (lossless1) [inline server/config/settings.py]: where's error log for file?

File: server/config/settings.py [hunk 2]
```diff
@@ -94,22 +94,26 @@
...
+	}
+	},
+	'loggers': {
+		'django': {
+			'handlers': ['file'],
+			'level': 'DEBUG',
+			'propagate': True,
 		},
```
Comment (lossless1) [inline server/config/settings.py]: and error log for console?

Comment (Mishawaka) [reply server/config/settings.py]: If I setup INFO level, it logs INFO and all after it (ERROR, WARNING, CRITICAL)

Comment (lossless1) [reply server/config/settings.py]: ok, got it

**Pull request merged**